mlflow/pyfunc/backend.py

```diff
@@ -73,7 +76,7 @@ def serve(self, model_uri, port, host):
         # NB: Absolute windows paths do not work with mlflow apis, use file uri to ensure
         # platform compatibility.
         local_uri = path_to_local_file_uri(local_path)
-        if os.name != "nt":
```

Comment:
`os.name != "nt"` is not the correct way to determine whether we are on Windows or not, as that value could be `"ce"` and could also be something else in some future version of Windows kernel